Packages/com.unity.inputsystem/InputSystem/Editor/InputEditorAnalytics.cs

```diff
@@ -1,924 +0,0 @@
-#if UNITY_EDITOR
```

Comment:
Good to see this bigger file split out. It looks like the split up has simplified the code and names the fields for more consistency with the backend analytics naming convention so it seems good. Hoping I haven't missed anything as this is a bigger change.

Reply:
I believe you did a non-full diff in GitHub which might have showed you something different than the actual end result but hope its ok :)

Reply:
I did just diff on changes since last review so perhaps it did get confused.

Reply:
I might have mixed a merge with some changes as well, I will resolve this